docs/api/en/math/Box3.html

```diff
@@ -12,11 +12,37 @@ <h1>[name]</h1>
 
 		<p class="desc">
 			Represents a box or cube in 3D space. The main purpose of this is to represent
-			the [link:https://en.wikipedia.org/wiki/Minimum_bounding_box Minimum Bounding Boxes]
-			for objects.
+			the world-axis-aligned bounding boxes for objects. 
 		</p>
 
+		
+		<h2>Example</h2>
 
+		<code>
+		// Creating the object whose bounding box we want to compute
+		var sphereGeom = new THREE.SphereGeometry();
+		var sphereObject = new THREE.Mesh( 
+			sphereGeom, 
+			new THREE.MeshBasicMaterial( 0xff0000 ) 
+		);
+
+		// Creating the actual bounding box with Box3
+		sphereObject.geometry.computeBoundingBox();
+		var BBox = new THREE.Box3(
+			sphereObject.geometry.boundingBox.min,
+  			sphereObject.geometry.boundingBox.max
+		);
```

Comment:
I don't understand why you pass in ctor paramters when you use the object for a copy operation two lines later. Besides `BBox` is no good variable name. `box` should be sufficient.

Reply:
Because the `copy` was supposed to be placed inside the animation loop. I'm editing the samples so that it is clear which part is the initialization and which is the updating. However, I think using clone makes the code sample more elegant and uniform so I'm gonna change it.